terraform/groups/identity-gateway/main.tf

```diff
@@ -0,0 +1,55 @@
+###
+# Data lookups
+###
+data "aws_vpc" "vpc" {
+  tags = {
+    Name = var.vpc_name
+  }
+}
+
+data "aws_subnet_ids" "data_subnets" {
+  vpc_id = data.aws_vpc.vpc.id
+  filter {
+    name   = "tag:Name"
+    values = ["*-applications-*"]
+  }
+}
+
+###
+# Modules
+###
+module "ecs" {
+  source       = "./modules/ecs"
+  service_name = var.service_name
+  vpc_id       = data.aws_vpc.vpc.id
+}
+
+module "webfiling_lb" {
+  source                = "./modules/loadbalancing"
+  service_name          = "forgerock-ig-webfiling"
+  vpc_id                = data.aws_vpc.vpc.id
+  vpn_cidrs             = values(data.terraform_remote_state.networking.outputs.vpn_cidrs)
+  subnet_ids            = data.aws_subnet_ids.data_subnets.ids
+  target_port           = 8080
+  domain_name           = var.domain_name
+  create_route53_record = var.create_route53_record
+  route53_zone          = var.route53_zone
+}
+
+module "webfiling" {
```

Comment:
Should this be 'webfiling' or 'webfiling-ig'?

Reply:
Doesn't matter much with the module name as this is just used within terraform. The service name is also `webfiling` though that is a Fargate service inside an ECS Cluster called `forgerock-identity-gateway` so it shouldn't cause any confusion